packages/create-block/lib/init-block.js

```diff
@@ -65,15 +68,17 @@ async function initBlockJSON( {
 
 module.exports = async function ( outputTemplates, view ) {
 	await Promise.all(
-		Object.keys( outputTemplates ).map(
-			async ( outputFile ) =>
-				await writeOutputTemplate(
-					outputTemplates[ outputFile ],
-					join( view.folderName, outputFile ),
-					view
-				)
-		)
-	);
+		Object.keys( outputTemplates ).map( async ( outputFile ) => {
+			const pathName = view.blockOnly
+				? join( process.cwd(), view.folderName, view.slug, outputFile )
```

Comment:
I quite like the idea of detecting the pacakage.json. This would also allow us to fire a warning to the user if there is not one found.

Reply:
> In case the block only flag, should we simply create a subdirectory named after `view.slug` with all templates? It would be the responsibility of the consumer to run the script in the correct place.

This would require them to move into the `src` ( or whatever dir name they are using ) directory, correct? It would be simpler for those at are using custom src dir names as it's not possible to pass the folder name to the CLI. It's automatically picked up from the template being used. Let's start here as an MVP and see what happens in the wild.